server/server.js

```diff
@@ -0,0 +1,54 @@
+import express from "express";
+import bodyParser from "body-parser";
+import path from "path";
+import { isCorrectAnswer, Questions, randomQuestion } from "./questions.js";
+import { set } from "husky";
+
+const app = express();
+app.use(bodyParser.json());
+
+let userAnswer;
+
+app.get("/api/question", (req, res) => {
+  const { question, id, answers } = randomQuestion();
+  console.log("Question:", question, "ID:", id, "answers:", answers);
+  res.json({ question, id, answers });
+});
+
+app.post("/api/question/answer", (req, res) => {
+  res.json("ok");
+  const { id, answer } = req.body;
+  const question = findQuestion(id);
+  console.log("id:", id, "answer:", answer);
+  function findQuestion(id) {
+    return Questions.find((q) => {
+      return q.id === id;
+    });
+  }
+  console.log("Find question:", findQuestion(id));
+
+  if (isCorrectAnswer(question, answer)) {
+    userAnswer = "correct";
+  } else {
+    userAnswer = "wrong";
+  }
+  console.log("outside:", userAnswer);
+});
+
+app.get("/api/question/result", (req, res) => {
```

Comment:
Du bør lagre score og andre ting i cookies istedet for en variabel som du gjør her. Se eksempler fra Johannes.